astro/src/content/docs/release-notes/__theme-update-warning.astro

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
not sure how this directory got off track but these files should only have a single underscore as a prefix

Reply:
Also these should really live in src/components/docs/release-notes

Reply:
Should they start with `_`?

Reply:
_underscore denotes a partial/fragment and Astro will not try to render those as their own page

Reply:
which is why this looks odd in this directory